index.js

```diff
@@ -73,7 +73,7 @@ function arg(opts, {argv = process.argv.slice(2), permissive = false, stopAtPosi
 
 			for (let j = 0; j < separatedArguments.length; j++) {
 				const arg = separatedArguments[j];
-				const [originalArgName, argStr] = arg[1] === '-' ? arg.split('=', 2) : [arg, undefined];
+				const [originalArgName, argStr] = arg[1] === '-' ? arg.split(/[=](.*)/, 2) : [arg, undefined];
```

Comment:
```suggestion
				const [originalArgName, argStr] = arg[1] === '-' ? arg.split(/=(.*)/, 2) : [arg, undefined];
```

Why the character class?

Reply:
Ah Just saw your other comment. We can disable that linting rule, we're all grownups here and can discern between division and regexes...

Reply:
Because there's a lint rule (probably from xo) that prevents using `/=(.*)/` and throws `no-div-regex` while running tests. I referred [no-div-regex](https://eslint.org/docs/rules/no-div-regex) for changes.

Reply:
Commenting to make sure the Github refresh bug didn't prevent you from seeing https://github.com/zeit/arg/pull/48#discussion_r302926360 @flexdinesh :)

Reply:
Should just be `{"xo": {"rules": {"no-div-regex": 0}}}` in `package.json` and then removing the `[]` around `[=]`.

Reply:
Lint rule removed and regex updated in a new commit